config/sys.config

```diff
@@ -1,6 +1,17 @@
 %% -*- erlang -*-
 [
- {blockchain_follower, []},
+ {blockchain_follower,
+  [
+   {follower_modules, []},
+   {ledger_required, false},
+   {sync_mode, false}
```

Comment:
Would suggest the behaviour require a sync() -> boolean callback to determine if the follower module needs to be sync, like the gen_statem:callback_mode()

Reply:
Except that we don't have a _single_ follower module we have a list of them. We _could_ just have a single follower callback module instead of a list of them, that makes it easier here and etl/node already support that approach too?

Reply:
right but you can decide if *any* are sync, and then make the decision.

`lists:foldl(fun(Module, Acc) -> Acc orelse Module:sync() end, false, Modules)`

Reply:
and what would we do for height?

Reply:
max or first reported height would be one option

Reply:
yeah, that seems like it'd work.

Reply:
it's not a terrible idea to run each follower in its own module though, as the events can be processed concurrently (other than the sync one).

Reply:
That makes me want to just support a single callback module and have the downstream implementations deal with how to dispatch (in)dependent loads?

Reply:
And then you can fire up as many bf_followers as you want.. That make sense?

Reply:
fine by me